gapir/cc/interpreter.cpp

```diff
@@ -107,12 +107,32 @@ void Interpreter::resetInstructions() {
   mCurrentInstruction = 0;
 }
 
+void Interpreter::buildJumpTable() {
```

Comment:
I would prefer to see this done on-demand. (I.e. when we hit a jump label, add it to the jump table, if we hit a jump that is not in the jump table, then scan forward until we find the jump label).

That way for the (more common case) where we don't have a jump table, we don't have to parse every opcode first.

Reply:
Even if we have 100mb of opcodes, I'd expect this to take a few tens of miliseconds. Is the extra complexity worth it for a speed-up that is only theoretical?

Reply:
Done it anyway!

Reply:
heh, thanks!
It practice it probably does not matter terribly much, but it just struck me as a future potentially irritating thing.